platform.mk

```diff
@@ -150,3 +150,11 @@ PRODUCT_PROPERTY_OVERRIDES += \
 PRODUCT_PROPERTY_OVERRIDES += \
     sys.usb.controller=a800000.dwc3 \
     sys.usb.rndis.func.name=gsi
+
+# verity
+PRODUCT_COPY_FILES += \
+    frameworks/native/data/etc/android.software.verified_boot.xml:system/etc/permissions/android.software.verified_boot.xml
```

Comment:
I think it must be moved to vendor

Reply:
why?

putting the xml file under `system/etc/permissions` is how it is done on bullhead and other devices:

```bash
android-8/device$ mgrep verified_boot

./google/dragon/device.mk:102:    frameworks/native/data/etc/android.software.verified_boot.xml:system/etc/permissions/android.software.verified_boot.xml \
./huawei/angler/device.mk:152:    frameworks/native/data/etc/android.software.verified_boot.xml:system/etc/permissions/android.software.verified_boot.xml \
./lge/bullhead/device.mk:120:    frameworks/native/data/etc/android.software.verified_boot.xml:system/etc/permissions/android.software.verified_boot.xml \
```